api/specs.proto

```diff
@@ -258,6 +258,12 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+
+	message SecurityConfig {
+	    string userns = 30;
```

Comment:
Ya I should have brought that comment here.  In the Engine PR I mention that I would like to also add `--capabilities` and a few others to this as well as anything else that other think makes sense.